data_prep/scotland/src/bin/generate_prioritization.rs

```diff
@@ -115,8 +122,37 @@ struct Stats19Input {
 impl Stats19Input {
     fn read_all_from_file() -> Result<Vec<Self>> {
         Ok(geojson::de::deserialize_feature_collection_str_to_vec(
-            &std::fs::read_to_string("atip-data-prep/stats19/tmp/stats19_clipped.geojson")
-                .expect("missing stats19_clipped.geojson - see get_reference_layers.sh"),
+            &fs_err::read_to_string("atip-data-prep/stats19/tmp/stats19_clipped.geojson")?,
         )?)
     }
 }
+
+#[derive(Deserialize)]
+struct InputPOI {
+    #[serde(deserialize_with = "geojson::de::deserialize_geometry")]
+    pub geometry: Point,
+    pub name: Option<String>,
+}
+
+impl InputPOI {
+    fn read_all_from_files() -> Result<Vec<POI>> {
+        let mut pois = Vec::new();
+        for (path, kind) in [
+            ("tmp/gp_practices.geojson", POIKind::GP),
+            ("tmp/hospitals.geojson", POIKind::Hospital),
+            ("tmp/schools.geojson", POIKind::School),
+        ] {
+            let input: Vec<InputPOI> = geojson::de::deserialize_feature_collection_str_to_vec(
+                &fs_err::read_to_string(path)?,
+            )?;
+            for poi in input {
+                pois.push(POI {
+                    kind,
+                    point: poi.geometry,
+                    name: poi.name,
```

Comment:
Nit: We used to show the number of pupils per school. Removed here because I'm not sure it actually matters. We could form a description string instead of name if it is important.